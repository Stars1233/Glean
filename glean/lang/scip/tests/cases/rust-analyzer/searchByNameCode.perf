{
  "@generated": null,
  "facts_searched": {
    "scip.Definition.1": 2,
    "scip.DisplayName.1": 2,
    "scip.DisplayNameSymbol.1": 2,
    "scip.FileLanguage.1": 1,
    "scip.SymbolKind.1": 2,
    "scip.SymbolName.1": 1
  },
  "full_scans": [ ]
}