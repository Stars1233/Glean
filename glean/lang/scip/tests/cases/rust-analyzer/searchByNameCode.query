query: codemarkup.search.SearchByName { searchcase = Sensitive, name = "Prefix" }
perf: True
transform: [gensort, []]
