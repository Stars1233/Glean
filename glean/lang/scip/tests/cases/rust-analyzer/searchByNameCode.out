[
  "@generated",
  {
    "key": {
      "searchcase": 0,
      "name": "Prefix",
      "entity": {
        "scip": {
          "rust": {
            "defn": {
              "key": {
                "symbol": {
                  "key": "rust-analyzer cargo relative_path 0.0.0 Prefix#"
                },
                "location": {
                  "key": {
                    "file": { "key": "relative_path.rs" },
                    "range": {
                      "lineBegin": 23,
                      "columnBegin": 10,
                      "lineEnd": 23,
                      "columnEnd": 15
                    }
                  }
                }
              }
            }
          }
        }
      },
      "location": {
        "name": "Prefix",
        "file": { "key": "relative_path.rs" },
        "location": {
          "range": {
            "file": { "key": "relative_path.rs" },
            "lineBegin": 23,
            "columnBegin": 10,
            "lineEnd": 23,
            "columnEnd": 15
          }
        }
      },
      "kind": 6,
      "language": 7
    }
  }
]